database/migration/V20170204_1037__createEventsTable.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `events` (
```

Comment:
The three migrations could probably be put in one file called `createEventsAPI.sql`, there also isn't a corresponding revert file for this migration.